### PR TITLE
[FUSEQE-5449] Batch Update automated test for DB

### PR DESCRIPTION
**Pull request by tplevko**
//test: `@webhook-db-batch`

### CODE REVIEW

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

File: ui-tests/src/test/resources/features/integrations/webhook-db.feature [hunk 1]
```diff
@@ -67,3 +67,51 @@ Feature: Integration - Webhook to DB
...
+    And click on the "Done" button
+    And publish integration
+    And set integration name "DB batch Insert"
+    And publish integration
+    Then wait until integration "DB batch Insert" gets into "Running" state
+
+    And select the "DB batch Insert" integration
+    And invoke post request to webhook in integration db-batch-insert with token test-webhook and body [{"task": "write doc", "completed": 1},{"task": "publish doc", "completed": 0}]
+
+    And check that query "SELECT * FROM TODO WHERE task = 'write doc'" has some output
```
Comment (mkralik3) [inline ui-tests/src/test/resources/features/integrations/webhook-db.feature]: Does the `clean application state` step reset a TODO table? If not, the TODO table should be resetted during the Background stage like the CONTACT table.

Comment (tplevko) [reply ui-tests/src/test/resources/features/integrations/webhook-db.feature]: You are right, good catch! Should be fixed now

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Pull request merged**